repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/string/Base64Helper.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+
+package org.openrepose.commons.utils.string
+
+import java.util.Base64
+import java.nio.charset.StandardCharsets.UTF_8
+
+object Base64Helper {
+  def base64EncodeUtf8(encodeString: String): String = {
+    new String(Base64.getEncoder.encodeToString(encodeString.getBytes(UTF_8)))
```

Comment:
I assume you'll drop the commit that brings this in and rebase if the other PR gets merged first, in which case this will be a collision. Or, if we keep this one, then you don't need the `new String` again.

Reply:
This was `cherry-pick`'d so should not collide and should be fine either way. The other branch already has the new commit to update this.

Reply:
Derp. Fair enough. For some reason, I was thinking that commit was modified rather than being two separate commits.

Reply:
I thought about it, but I knew I had `cherry-pick`'d it and didn't want to deal with the conflicts unless I had to.